crates/iceberg/src/spec/manifest/entry.rs

```diff
@@ -533,6 +569,8 @@ fn data_file_fields_v2(partition_type: &StructType) -> Vec<NestedFieldRef> {
         SORT_ORDER_ID.clone(),
         FIRST_ROW_ID.clone(),
         REFERENCE_DATA_FILE.clone(),
+        // Why are the following two fields here in the existing v2 schema?
+        // In the spec, they are not even listed as optional for v2.
```

Comment:
I have the same doubt and this question also applies to the `FIRST_ROW_ID` above